### PR TITLE
Fix #1904: Enable more blacklisted partests

### LINKED ISSUE #1904: Enable more blacklisted partests
A quick run of blacklisted partests shows, that there are over an estimated 5% (13/172) that can be enabled since they either just pass, or only need a checkfile change.

I'll run this over night so we can enable them.

### PR DESCRIPTION
**Pull request by gzm0**

### CODE REVIEW

**Comment by gzm0** [comment]
Review by @sjrd

**Comment by gzm0** [comment]
The initial partest list to be added (before filtering for too slow tests etc.) originates from the 2.11.7 partest. All the partests newly added to the whitelists have been tested in 2.11.7, all three optimization modes.

File: partest-suite/src/test/resources/scala/tools/partest/scalajs/2.11.0/WhitelistedTests.txt [hunk 1]
```diff
@@ -2910,9 +2933,15 @@ run/misc.scala
...
+neg/t6446-show-phases.scala
+neg/t6446-additional
 
 # Adapt checkfiles for different behavior with boxed types
 run/t5568.scala
 run/virtpatmat_typetag.scala
 run/virtpatmat_switch.scala
 run/t5629b.scala
+run/t5356.scala
+
```
Comment (sjrd) [inline partest-suite/src/test/resources/scala/tools/partest/scalajs/2.11.0/WhitelistedTests.txt]: Double EOL at EOF

File: partest-suite/src/test/resources/scala/tools/partest/scalajs/2.11.0/neg/t6446-list.check [hunk 2]
```diff
@@ -0,0 +1,2 @@
+ploogin - A sample plugin for testing.
+scalajs - Compile to JavaScript
```
Comment (sjrd) [inline partest-suite/src/test/resources/scala/tools/partest/scalajs/2.11.0/neg/t6446-list.check]: Missing EOL at EOF

File: partest-suite/src/test/resources/scala/tools/partest/scalajs/2.11.1/neg/t6446-list.check [hunk 3]
```diff
@@ -0,0 +1,2 @@
+ploogin - A sample plugin for testing.
+scalajs - Compile to JavaScript
```
Comment (sjrd) [inline partest-suite/src/test/resources/scala/tools/partest/scalajs/2.11.1/neg/t6446-list.check]: Same

File: partest-suite/src/test/resources/scala/tools/partest/scalajs/2.11.2/neg/t6446-list.check [hunk 4]
```diff
@@ -0,0 +1,2 @@
+ploogin - A sample plugin for testing.
+scalajs - Compile to JavaScript
```
Comment (sjrd) [inline partest-suite/src/test/resources/scala/tools/partest/scalajs/2.11.2/neg/t6446-list.check]: Same

File: partest-suite/src/test/resources/scala/tools/partest/scalajs/2.11.5/neg/t6446-list.check [hunk 5]
```diff
@@ -0,0 +1,2 @@
+ploogin - A sample plugin for testing.
+scalajs - Compile to JavaScript
```
Comment (sjrd) [inline partest-suite/src/test/resources/scala/tools/partest/scalajs/2.11.5/neg/t6446-list.check]: Same

File: partest-suite/src/test/resources/scala/tools/partest/scalajs/2.11.6/neg/t6446-list.check [hunk 6]
```diff
@@ -0,0 +1,2 @@
+ploogin - A sample plugin for testing.
+scalajs - Compile to JavaScript
```
Comment (sjrd) [inline partest-suite/src/test/resources/scala/tools/partest/scalajs/2.11.6/neg/t6446-list.check]: Same

File: partest-suite/src/test/resources/scala/tools/partest/scalajs/2.11.7/neg/t6446-list.check [hunk 7]
```diff
@@ -0,0 +1,2 @@
+ploogin - A sample plugin for testing.
+scalajs - Compile to JavaScript
```
Comment (sjrd) [inline partest-suite/src/test/resources/scala/tools/partest/scalajs/2.11.7/neg/t6446-list.check]: Same

File: partest-suite/src/test/resources/scala/tools/partest/scalajs/2.11.7/run/delambdafy-specialized.check [hunk 8]
```diff
@@ -0,0 +1 @@
+scala.runtime.AbstractFunction1
```
Comment (sjrd) [inline partest-suite/src/test/resources/scala/tools/partest/scalajs/2.11.7/run/delambdafy-specialized.check]: Same

File: partest-suite/src/test/resources/scala/tools/partest/scalajs/2.12.0-M3/WhitelistedTests.txt [hunk 9]
```diff
@@ -3135,3 +3162,5 @@ run/virtpatmat_typetag.scala
 run/virtpatmat_switch.scala
 run/t5629b.scala
 run/t6318_primitives.scala
+run/t5356.scala
+
```
Comment (sjrd) [inline partest-suite/src/test/resources/scala/tools/partest/scalajs/2.12.0-M3/WhitelistedTests.txt]: Double EOL at EOF

File: partest-suite/src/test/resources/scala/tools/partest/scalajs/2.12.0-M3/neg/t6446-list.check [hunk 10]
```diff
@@ -0,0 +1,2 @@
+ploogin - A sample plugin for testing.
+scalajs - Compile to JavaScript
```
Comment (sjrd) [inline partest-suite/src/test/resources/scala/tools/partest/scalajs/2.12.0-M3/neg/t6446-list.check]: Missing EOL at EOF

**Comment by sjrd** [comment]
That's all.

**Comment by gzm0** [comment]
Updated

**Comment by sjrd** [comment]
LGTM

**Comment by sjrd** [comment]
Nightly: https://scala-webapps.epfl.ch/jenkins/job/scalajs-matrix-build/3006/

**Comment by gzm0** [comment]
Nightly passed. Merging

**Comment by sjrd** [comment]
Waw, a nightly that actually passed ^^

**Pull request merged**